frontend/components/SponsorModal.tsx

```diff
@@ -24,25 +26,27 @@ const SponsorsModal = ({ handleClose, sponsorName, sponsors }: any) => {
           src={
             window.matchMedia &&
             window.matchMedia('(prefers-color-scheme: dark)').matches
-              ? `/sponsors/2022/dark-mode/${sponsors[sponsorName][2]}`
-              : `/sponsors/2022/${sponsors[sponsorName][2]}`
+              ? 'https:' + sponsor.fields.darkModeLogo.fields.file.url
+              : 'https:' + sponsor.fields.lightModeLogo.fields.file.url
           }
-          alt={sponsorName}
+          alt={sponsor.fields.name}
         />
-        <h1 className={styles.sponsorType}>
-          {sponsors[sponsorName][3]} SPONSOR
-        </h1>
+        <h1 className={styles.sponsorType}>{sponsorType.slice(0, -1)}</h1>
         <div className={styles.inLine}>
           <p className={styles.sponsorWebsite}>Website: </p>
           <Link
             target="blank"
-            href={sponsors[sponsorName][1]}
+            href={sponsor.fields.website}
             rel="noopener noreferrer"
           >
-            <a className={styles.sponsorLink}>{sponsors[sponsorName][1]}</a>
+            <a className={styles.sponsorLink}>{sponsor.fields.website}</a>
           </Link>
         </div>
-        <p className={styles.sponsorDescription}>{sponsors[sponsorName][4]} </p>
+        {Object.keys(content).map((contentNo, index) => (
+          <p key={index} className={styles.sponsorDescription}>
```

Comment:
Unsure if this is intended behaviour, but the descriptions on dark mode don't seem to appear? 
<img width="1491" alt="image" src="https://user-images.githubusercontent.com/80564580/209900644-02845bda-0a31-4eea-b18d-2227460d2660.png">